drivetrain.py

```diff
@@ -137,7 +146,7 @@ def stop(self):
 
 
     def lights(self, on):
-        rgb_values = self.FULL_WHITE if on else self.OFF
+        rgb_values = self.colours.get('YELLOWISH') if on else self.OFF
```

Comment:
Minor: It is a good idea to use the optional second argument to `get()` in order to provide a default value to be returned in the event that the requested key is not present. The default bahaviour is to return `None`, which can cause exceptions later on unless explicitly handled.

[docs](https://docs.python.org/2/library/stdtypes.html#dict.get)